.github/workflows/automerge.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
           GITHUB_LOGIN: nsmbot
           ENABLED_FOR_MANUAL_CHANGES: true
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          MERGE_METHOD: REBASE
```

Comment:
Please remove this.

It will fail automerge.

Reply:
Done.